tests/unit/translator/resources/for_with_range.pr

```diff
@@ -48,6 +48,12 @@ process Taskwithloop;
             task for x in RANGE(0, 10, 2):
 y := y + x
 y := Y * 2
+endfor;
+            task for x in RANGE(1, 3):
+y := y + x
+endfor;
+            task for x in RANGE(4):
```

Comment:
Can we test in RANGE(length(somearray))?

Reply:
added